moeflow_worker.py

```
@@ -0,0 +1,197 @@
+            return {
+                'pts': o.pts,
+                'text': o.text,
+                'prob': o.prob,
+                'textlines': self.default(o.textlines),
+            }
+        elif isinstance(o, filter) or isinstance(o, tuple):
+            return self.default(list(o))
+        elif isinstance(o, list):
+            return o
+        elif isinstance(o, str):
+            return o
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)